pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -221,10 +227,26 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	pa.Status.InitializeConditions()
 	logger.Debug("PA exists")
 
-	if err := c.reconcileMetricsService(ctx, pa); err != nil {
+	scale, err := c.scaler.GetScaleResource(pa)
+	if err != nil {
+		logger.Debugw("Error Getting scale", zap.Error(err))
```

Comment:
Should this be `Errorw`? Do we need it at all given we return an error with context? (Applies to all error logs in this PR I think)

Reply:
Not really, since the error would be logged as error in the common controller code. So here it's mostly for local debugging, which logs the debug level.